.CI/CreateUbuntuDeb.sh

```diff
@@ -42,8 +42,15 @@ fi
 
 chatterino_version=$(git describe 2>/dev/null | cut -c 2-) || true
 if [ -z "$chatterino_version" ]; then
-    # Fall back to this in case the build happened outside of a git repo
+    # Fall back to this in case the build happened outside of a git repo or a repo without tags
     chatterino_version="0.0.0-dev"
+elif [ "$chatterino_version" = "ightly-build" ]; then
```

Comment:
```suggestion
elif [ "$chatterino_version" = "nightly-build" ]; then
```

Reply:
That is because the earlier call uses `cut -c 2-` which removes the first char, so it is intentional that the `n` is missing.

Reply:
I don't think we need to handle the case where `chatterino-version` is `nightly-build`, it's only ever relevant for forks that have no tags.

If you want to improve this, I'd rather it be done in a separate PR so we can nitpick over it without blocking the main purpose of the PR